sdk/servicebus/service-bus/test/internal/tracing.spec.ts

```diff
@@ -125,7 +127,7 @@ describe("Tracing tests", () => {
     receiver.subscribe(
       {
         processMessage: async (msg) => {
-          if (msg.properties!["Diagnostic-Id"] === "should throw") {
+          if (msg.applicationProperties!["Diagnostic-Id"] === "should throw") {
```

Comment:
Certain tests in this file are failing. Run `rushx unit-test:node` in the service-bus folder to see the failures. It is most likely because the stubs are still using `properties` instead of `applicationProperties`